### PR TITLE
SOEOP-366 - updated styles per feedback.

### PR DESCRIPTION
**Pull request by mdyoung3**
# READY FOR REVIEW

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
12/3

# Criticality
- 3/10

# Review Tasks

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOEOP-366

### CODE REVIEW

File: scss/components/_soe_news.scss [hunk 1]
```diff
@@ -99,11 +99,15 @@
   }
 }
 
+div#block-ds-extras-banner {
```
Comment (sherakama) [inline scss/components/_soe_news.scss]: It is discouraged to combine element and id/class selectors. The `#` selector should be plenty strong enough.

File: css/soe_helper.css [hunk 2]
```diff
@@ -1595,11 +1595,14 @@ body:not(.page-admin) .sidebar .views-exposed-form .views-exposed-widget .form-s
   .node-type-stanford-news-item .group-s-credits-style.field-group-div.credits {
     display: block; } }
 
+div#block-ds-extras-banner {
```
Comment (sherakama) [inline css/soe_helper.css]: Don't combine element and ID selectors.
```suggestion
#block-ds-extras-banner {
```

File: css/soe_helper.css [hunk 3]
```diff
@@ -1595,11 +1595,14 @@ body:not(.page-admin) .sidebar .views-exposed-form .views-exposed-widget .form-s
...
+  margin-bottom: -70px; }
```
Comment (sherakama) [inline css/soe_helper.css:1602]: That's a whole lotta negative margin. Validate that this is good on all screen sizes please.

Comment (mdyoung3) [reply css/soe_helper.css:1599]: Screen Recording 2020-12-04 at 11 00 20 AM

Comment (mdyoung3) [reply css/soe_helper.css:1602]: Good call. Had to make some changes for mobile. It looks good now. 

Screen Recording 2020-12-04 at 12 27 16 PM

Comment (mdyoung3) [reply css/soe_helper.css:1602]: Did another update. Now it's even smoother. 

Screen Recording 2020-12-04 at 2 23 08 PM

File: scss/components/_soe_news.scss [hunk 4]
```diff
@@ -99,11 +99,19 @@
   }
 }
 
+#block-ds-extras-banner {
+  margin-bottom: -70px;
+
+  @media (max-width: 485px) {
```
Comment (sherakama) [inline scss/components/_soe_news.scss:105]: Is there a breakpoint with this pixel dimension? Shouldn't it use one of the mixins?

eg:
```
@include breakpoint-max(med-small) {
    margin-bottom: 0;
}
```

Comment (mdyoung3) [reply scss/components/_soe_news.scss:105]: I don't think there is. Which is why it looking janky.

**Review by sherakama — APPROVED** [review]
Ok, by me. @boznik's turn.

**Review by boznik — APPROVED** [review]
Code seems fine, if the visual outcome has been reviewed, it's good by me.

**Comment by mdyoung3** [comment]
Thank you @boznik and @sherakama for jumping on this.

**Pull request merged**